.gitignore

```diff
@@ -0,0 +1,2 @@
+packer_cache
+output-virtualbox-iso
```

Comment:
It's there. Github doesn't seem to like to show it. See https://github.com/pulsesecure/vagrant-alpine-3.3-x86_64-docker/blob/a2ece8c32ae88a0bac0fa261a37c1041e9a68365/.gitignore says "3 lines (2 sloc)"